atlas-slurm

```diff
@@ -128,7 +128,7 @@ else
             logFile=${logPrefix}.out
         fi 
 	sleep 10
-	slurm_monitor_job "$slurmJobId" "$pollFreqSecs" "${workingDir}/${logFile}" "$monitorStyle" "$logCleanup" "$returnStdout" "$quiet"
+	slurm_monitor_job "$slurmJobId" "$pollFreqSecs" "$logFile" "$monitorStyle" "$logCleanup" "$returnStdout" "$quiet"
```

Comment:
have you tested with this? It may fail or throw error

Reply:
especially when just file name is specified not path

Reply:
So the thing is, in `slurm_monitor_job` the basename of `logFile` should be consistent with the `logPrefix` given in `slurm_submit` [here](https://github.com/ebi-gene-expression-group/atlas-bash-util/blob/2102391f8f986f73bb85b127a76f83adf1f4e760/atlas-slurm#L117).  The change above makes them consistent.  The log is written in the supplied log prefix (no filename is supplied by the user, only a prefix).  If there is no path, the log is written in the present working dir (the supplied `workingDir`), as is the behavior in `atlas-lsf`.